package com.itshare.exceptions.example2;

public class Calculator {

	
	public int divide(int num1 , int num2) {
		
		ArithmeticException e = new ArithmeticException();
		ArithmeticException e = new ArithmeticException("this and exception");

		
		if(num2 == 0)
		throw new ArithmeticException("This is exception by 0"); 
		
		return num1/num2;
	}
	
}
